bangazonapi/views/profile.py

```diff
@@ -371,11 +372,12 @@ class ProfileSerializer(serializers.ModelSerializer):
     """
     user = UserSerializer(many=False)
     recommends = RecommenderSerializer(many=True)
+    recommended = RecommenderSerializer(many=True)
```

Comment:
You'll want to either make a new serializer or add the `recommender` field to the `RecommendationSerializer`